Solutions/RedCanary/Analytic Rules/RedCanaryThreatDetection.yaml

```diff
@@ -0,0 +1,123 @@
+id: 6d263abb-6445-45cc-93e9-c593d3d77b89
+kind: Scheduled
+name: Red Canary Threat Detection
+description: Triggers Incidents using detection data assembled by Red Canary.
+severity: High
+requiredDataConnectors:
+  - connectorId: RedCanaryDataConnector
+    dataTypes:
+      - RedCanaryDetections_CL
+queryPeriod: 5m
+queryFrequency: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Collection
+  - CommandAndControl
+  - CredentialAccess
+  - DefenseEvasion
+  - Discovery
+  - Execution
+  - Exfiltration
+  - Impact
+  - InitialAccess
+  - LateralMovement
+  - Persistence
+  - PrivilegeEscalation
+  - PreAttack
```

Comment:
PreAttack is not currently a supported tactics for the Sentinel queries. Here is a list that we currently support: https://attack.mitre.org/tactics/enterprise/ 

Please remove this from the tactics list.